apps/web/components/defined-term/DefinedTerm.tsx

```diff
@@ -34,6 +34,7 @@ export default function DefinedTerm({
 
   const tooltipButton = (
     <Tooltip
+      tooltipLabel={`${term} select to open glossary modal`}
```

Comment:
It's currently reading the term twice, once for the button text and once for this insertion in the label. I'd also probably call it the 'defined terms' modal as that is the title in the  modal and what the building code calls it.